english/find-exit/into-forest/forest.md

```diff
@@ -17,4 +17,16 @@ Version control rules
 
 Managed projects is the aim
 
-Peace of mind the gain.
\ No newline at end of file
+Peace of mind the gain.
+
+But only for a short while
+
+That something watching you emerges from the shadows
+
+It's a cougar, stalking it's prey
+
+It approaches you, and you can do nothing but wait for it to make it's move
+
+Then it asks "What's your name?"
+
+[Do you start a conversation with the cougar?] (https://github.com/zmorris1/create-your-own-adventure/blob/talk-to-cougar/english/find-exit/into-forest/talk-to-cougar/talk-to-cougar.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.